backend/common_python/server_common.py

```
@@ -283,44 +403,26 @@ def run_zmq_server(socket: zmq.Socket, handler: Any) -> None:
-                    msgpack_encoder.encode(r) for r in resps
-                ]
-                socket.send_multipart(response_msg)
-
-    except (zmq.ZMQError, OSError, ValueError, RuntimeError, KeyError) as exc:
-        print(
-            f"\n[!!!] Unhandled error occurred in the ZMQ server loop: {exc}",
-            file=sys.stderr,
-        )
-        traceback.print_exc()
-        os._exit(1)
+    except Exception as exc:  # pylint: disable=broad-exception-caught
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)